CONTRIBUTING.md

```diff
@@ -28,7 +28,8 @@ If you're submitting a **pull request** (bugfix/addition):
 
 ## Making Changes
 
-- Before making any changes, refer to the [DuckDuckHack Styleguide](https://dukgo.com/duckduckhack/styleguide_overview) to ensure your changes are made in the correct fashion
+- Before making any changes, refer to the [DuckDuckHack Styleguide](https://duck.co
+- /duckduckhack/styleguide_overview) to ensure your changes are made in the correct fashion
```

Comment:
@javathunderman Theres an added space after `duck.co` see the display on this [page](https://github.com/javathunderman/zeroclickinfo-spice/blob/patch-1/CONTRIBUTING.md)

![selection_233](https://cloud.githubusercontent.com/assets/5282264/6214249/02ed848a-b632-11e4-9ccc-e73d1d8d2bd9.png)